.github/workflows/docker.yml

```diff
@@ -1,9 +1,17 @@
-# This workflow will only build for Pull Requests which are opened, changed , merged 
-name: Docker build
-on:
-  pull_request:
-    types: [opened, synchronize, reopened]
-
-jobs:
-  docker:
-    uses: usdot-fhwa-stol/actions/.github/workflows/docker.yml@main
\ No newline at end of file
+# This workflow will only build for Pull Requests which are opened, changed , merged 
+name: Docker build
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  # Jammy 22.04 (ROS2 Humble)
+  docker-build-jammy:
+    uses: usdot-fhwa-stol/actions/.github/workflows/docker.yml@main

```

Comment:
Are the two jobs triggered when changes are pushed ? May be can we use this path filters (https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpaths) to only run specific job that is required when its associated files changed instead of running multiple jobs every time ?

Reply:
Ah yes great point I forgot. I tried adding those. I had to separate the yml files to write it cleanly.

Reply:
I tested the change by creating a [dummy PR here](https://github.com/usdot-fhwa-stol/carma-base/pull/192) that points to this branch. The dummy PR only has jammy changes, which successfully only triggered jammy docker: https://github.com/usdot-fhwa-stol/carma-base/actions/runs/11247485262/job/31271050955?pr=192